src/styles/03-organisms/list/_list.scss

```diff
@@ -0,0 +1,10 @@
+.unordered-list {
```

Comment:
I'd move this to `atoms/text/_text.scss`, unordered lists are just inline elements, feels weird to have 'em all the way up in `organisms/`